dtm0/fop.c

```diff
@@ -267,6 +277,9 @@ static int dtm0_fom_create(struct m0_fop *fop,
 	} else if (req->dtr_msg == DTM_TEST) {
 		m0_fom_init(&fom->dtf_fom, &fop->f_type->ft_fom_type,
 			    &dtm0_tmsg_fom_ops, fop, repfop, reqh);
+	} else if (req->dtr_msg == DTM_NET) {
```

Comment:
I don't understand why some message type 'NET' is needed.
We already have P message, REDO message.
Is that enough?